crypto/ec/build.info

```diff
@@ -55,7 +53,7 @@ $COMMON=ec_lib.c ecp_smpl.c ecp_mont.c ecp_nist.c ec_cvt.c ec_mult.c \
         curve448/curve448_tables.c curve448/eddsa.c curve448/curve448.c \
         $ECASM
 SOURCE[../../libcrypto]=$COMMON ec_ameth.c ec_pmeth.c ecx_meth.c ec_err.c \
-                        ecdh_kdf.c eck_prn.c
+                        ecdh_kdf.c eck_prn.c ec_evp_lib.c
```

Comment:
For RSA `crypto/rsa/rsa_lib.c` contains this stuff, but that is not an option for `crypto/ec/ec_lib.c` as that needs to be pulled in both the default and the fips provider.